packages/components/src/DateTimePickers/Time/Manager/Manager.component.js

```diff
@@ -1,84 +1,78 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 
 import { TimeContext } from '../Context';
 import extractTime, { getTimeFormat } from '../time-extraction';
 
-class ContextualManager extends React.Component {
-	static displayName = 'Time.Manager';
-	static propTypes = {
-		children: PropTypes.node,
-		onChange: PropTypes.func,
-		useSeconds: PropTypes.bool,
-		timezone: PropTypes.string,
-		value: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
-	};
+function TimeContextualManager(props) {
+	const [state, setState] = useState(extractTime(props.value, props.useSeconds));
```

Comment:
The problem: `extractTime` is executed at each render, and passed to useState. But useState only uses it at first render.
To fix that, we can provide an initialisation function instead that will be called only at first render

```suggestion
	const [state, setState] = useState(() => extractTime(props.value, props.useSeconds));
```